WordPress/src/main/java/org/wordpress/android/ui/media/MediaGalleryPickerActivity.java

```diff
@@ -113,15 +121,12 @@ protected void onCreate(Bundle savedInstanceState) {
 
         if (mIsSelectOneItem) {
             mGridAdapter.setAllowMultiselect(false);
-            setTitle(R.string.select_from_media_library);
-            ActionBar actionBar = getSupportActionBar();
-            if (actionBar != null) {
-                actionBar.setDisplayHomeAsUpEnabled(true);
-            }
         } else {
             mGridAdapter.setAllowMultiselect(true);
-            mGridAdapter.setInMultiSelect(true);
-            mGridAdapter.setSelectedItems(selectedItems);
+            if (selectedItems != null && selectedItems.size() > 0) {
```

Comment:
Small nit: we shouldn't need the `null` check here, as we set up `selectedItems` as an empty list above and haven't done anything that could `null` it (this generates a lint warning).